service/gcs/runtime/runc/utils_test.go

```diff
@@ -147,11 +147,12 @@ var _ = Describe("Utils", func() {
 			expectedPath string
 			actualPath   string
 		)
+		id := "atestid"
 		BeforeEach(func() {
-			expectedPath = "/var/run/gcsrunc/log.log"
+			expectedPath = "/tmp/gcs/" + id + "runc.log"
```

Comment:
You'll probably notice this when the tests are done, but I think there should be a slash before runc.log here.

Reply:
Ha. Yea your right. I wish I could run these locally